Austin - Added helpmsg, updated main readme with UI

austin-currington opened the pull request:
I made a couple of related changes - I added a -help CLI flag which displays CLI flag usage and aborts. I also added some usage information on this and the GUI to the main readme.md file.

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Fix readme link

JacobSmith1020 committed:
Merge branch 'main' into austin

Commit:
Update main.py

Commit:
Update main.py

Fix whitespaces, add sys import I forgot

Commit:
Update main.py

Simplify cmd-line use existing embed flag

Commit:
revert embed change, add cmd test bat script for windows

andrew-fennell committed:
Merge main, moved cmdTestText.bat

Commit:
Fix text script syntax and add audio script

Commit:
Update cmdTestAudio.bat

Fix typo

Commit:
Add -help flag with usage

Added help message with description of flags and their values

Commit:
Update README.md

Wrote about usage of UI in readme.md

austin-currington commented:
Oops, looks like I have some lingering issues with the cmdTest files. They're out of date now, because of auto-language-detection. I'll need to update them first.

Commit:
Update scripts because of autoLanguage

I needed to remove the flags for specifying language now that it's automatic.

austin-currington commented:
I also want to add to the readme something about the windows context menu stuff, but I don't know how much detail to go into there.

austin-currington committed:
Merge branch 'main' into austin

andrew-fennell commented on CogNative/main.py:
```diff
@@ -17,6 +17,19 @@
+if '-help' in args:
+    print("""
+CogNative CLI FLags: 
+    -sampleAudio <PATH>: audio file of voice to copy
```
```suggestion
    -sampleAudio <PATH>: audio file of voice to clone
```

Maybe this change to further distance ```-sampleAudio``` from ```-dialogueAudio```.

austin-currington committed:
Update CogNative/main.py

Reword one part of helpmsg

Co-authored-by: Andrew Fennell <andrew.fennell1999@me.com>

austin-currington committed:
Update README.md

Change readme flag section to bring into line with rewording "copy" to "clone" for input sample

andrew-fennell reviewed (approved):
LGTM
